actionpack/test/controller/api/conditional_get_test.rb

```diff
@@ -3,6 +3,10 @@
 require "abstract_unit"
 require "active_support/core_ext/integer/time"
 require "active_support/core_ext/numeric/time"
+require "active_record"
```

Comment:
Action Pack tests can't depend on Active Record. If you really need to test this behavior in a controller you will need to write a test in the railtie folder.

Reply:
Thanks for the clarification. I've removed the Action Pack tests. (I'm ambivalent on whether there really needs to be a test for this. Now that `cache_key` and `cache_version` have been split into separate methods, I expect this interface will remain stable.)